src/ReactLeafletMap/layers/CanvasVectorLayer/LeafletCanvasLayer.js

```diff
@@ -45,7 +45,7 @@ const LeafletCanvasLayer = L.CanvasLayer.extend({
 			const self = this;
 
 			// TODO breakable loop?
-			this.features.forEach(feature => {
+			this.features?.forEach(feature => {
```

Comment:
Secure if features is null.

Reply:
4dd36b335958ec8c9d68eb519561ade5c80faa60